packages/backend-common/src/service/types.ts

```diff
@@ -98,6 +98,22 @@ export type ServiceBuilder = {
     requestLoggingHandler: RequestLoggingHandlerFactory,
   ): ServiceBuilder;
 
+  /**
+   * Set the error handler
+   *
+   * If no handler is given the default one is used
```

Comment:
This doc string needs updating right? 
Should be something like(feel free to word it as you'd like)

> Sets an additional errorHandler to run before the default error handler. To disable the default error handler you need to invoke disableDefaultErrorHandler()

Reply:
You are right, I've forgotten to change that, I've just pushed the changes